### PR TITLE
Release Prep for 5.3.1

### PR DESCRIPTION
**Pull request by tony-landreth**
Issue: [sc-17776]

### CODE REVIEW

File: helm/install/values.yaml [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
 relatedImages:
   postgres_15:
-    image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-15.1-0
+    image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-15.2-0
   postgres_15_gis_3.3:
-    image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis:ubi8-15.1-3.3-0
+    image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis:ubi8-15.2-3.3-0
   postgres_14:
-    image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.6-2
+    image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.7-2
```
Comment (tjmoore4) [inline helm/install/values.yaml:14]: :wrench: 
```suggestion
    image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.7-0
```

File: helm/postgres/values.yaml [hunk 2]
```diff
@@ -59,7 +59,7 @@ postgresVersion: 14
 
 # imageExporter is the image name for the exporter used as a part of monitoring.
 # This defaults to the value below.
-# imageExporter: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.3.0-0
+# imageExporter: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.3.1-0
```
Comment (tjmoore4) [inline helm/postgres/values.yaml:62]: :wrench: 
There are component versions above for postgres, pgbackrest and pgbouncer that need to be adjusted.

**Review by tjmoore4 — CHANGES_REQUESTED** [review]
A few tweak, but overall looks good.

File: helm/postgres/values.yaml (hunk 2 above)
Comment (tony-landreth) [reply helm/postgres/values.yaml:62]: Thanks for catching that!

**Review by tjmoore4 — APPROVED** [review]

**Comment by shusaan** [comment]
Hi
I just checked this version https://access.crunchydata.com/documentation/postgres-operator/v5/releases/5.3.1/
when it will be merged?

One more think is that this container image is not available 
`registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-13.10-1`

**Comment by sigwinch28** [comment]
Looking forward to this PR moving forwards 🙏

**Pull request merged**